CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Improve confirmation modal footer for `ControlledVocab` component. Refs STSMACOM-863.
 * Add the `endDateInputRef` prop to `DateRangeFilter` to access the end date element. Refs STSMACOM-859.
 * Remove unnecessary `aria-rowindex` in `ItemView` and `ItemEdit` components. Fixes STSMACOM-871.
+* Check for `active` status of `<Settings>` navigation links independently from the applie href and query string. Query string should only be applied to active link. Refs STSMACOM-837.
```

Comment:
"applie" is a typo I think? I don't understand what the buggy behavior was, or what the correct behavior is now. I don't think the technical details about href and query-string are important here; they can be in comments in the code or the PR description. Maybe this could be, "Correctly label active NavListItems in settings navigation"?